src/tests/unit/tests/injected/visualization/get-visualization-for-tab-stops.test.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { FeatureFlags } from 'common/feature-flags';
+import {
+    SingleTabStopRequirementState,
+    TabbedElementData,
+    TabStopRequirementInstance,
+    TabStopRequirementState,
+    TabStopsScanResultData,
+} from 'common/types/store-data/visualization-scan-result-data';
+import { SelectorToVisualizationMap } from 'injected/selector-to-visualization-map';
+import { GetVisualizationInstancesForTabStops } from 'injected/visualization/get-visualization-instances-for-tab-stops';
+
+describe('GetVisualizationInstancesForTabStops', () => {
+    let tabbedElements: TabbedElementData[];
+    let expectedResults: SelectorToVisualizationMap;
+
+    beforeEach(() => {
+        tabbedElements = [
+            {
+                target: ['some', 'target'],
+                tabOrder: 1,
+                timestamp: 123,
+            },
+            {
+                target: ['another', 'target'],
+                tabOrder: 2,
+                timestamp: 124,
+            },
+        ] as TabbedElementData[];
+        expectedResults = {
+            'some;target': buildVisualizationInstance(tabbedElements[0].target, false, {
+                tabOrder: tabbedElements[0].tabOrder,
+                timestamp: tabbedElements[0].timestamp,
+            }),
+            'another;target': buildVisualizationInstance(tabbedElements[1].target, false, {
+                tabOrder: tabbedElements[1].tabOrder,
+                timestamp: tabbedElements[1].timestamp,
+            }),
+        } as SelectorToVisualizationMap;
+    });
+
+    test('GetVisualizationInstancesForTabStops: without requirement instances with feature flag off', () => {
```

Comment:
nit: I found these test names confusing, as it seems to imply a testing matrix with 4 possibilities (requirement instances on/off, feature flag on/off). Can we either add those other tests (with requirement instances and feature flag off/ without requirement instances and feature flag on) or alter these test names slightly?